integration/matchers/flux.go

```diff
@@ -4,18 +4,18 @@ package matchers
 
 import (
 	"fmt"
-	"io/ioutil"
-	"os"
-	"os/exec"
-	"path/filepath"
-	"github.com/weaveworks/eksctl/integration/utilities/git"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	"github.com/weaveworks/eksctl/integration/utilities/git"
```

Comment:
seems like you are sorting imports, can you run `goimports -w <file path>` for this file instead ?

Reply:
fixed, and `--force-push`ed.